### ISSUE TITLE
Create the MET-11.1.0-beta2 release

### ISSUE THREAD

**Issue by JohnHalleyGotway** [dtcenter/MET#2517]
## Describe the Task ##

Create the MET-11.1.0-beta2 development release. Remember to create a new discussion for testing the beta2 release, as we did for beta1 with dtcenter/METplus#2039.

### Time Estimate ###
2 hours

### Sub-Issues ###
Consider breaking the task down into sub-issues.
None needed.

### Relevant Deadlines ###
Should be created on 2/28/23.

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED PROJECT ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)
No impacts.

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

**Comment by JohnHalleyGotway** [comment]
Reopening this issue on 4/28/23 since we're recreating this beta2 release to include one more fix.

PR #2509 for issue #2285 caused unexpected differences in the METplus use cases. Issue #2525 fixes the underlying problem in MET. Recreating beta2 to include it.

**Comment by JohnHalleyGotway** [comment]
Merging PR #2529 for issue #2525 triggered [this GHA run](https://github.com/dtcenter/METplus/actions/runs/4832165717) in METplus. One difference is flagged in a short-range use case test:
```
ERROR: Some differences were found
NetCDF diff
A:/data/truth/short_range/GenEnsProd_fcstHRRR_fcstOnly_SurrogateSevere/short_range/surrogate_severe_calc/surrogate_severe_20200205_036V_regrid.nc
B:/data/output/short_range/GenEnsProd_fcstHRRR_fcstOnly_SurrogateSevere/short_range/surrogate_severe_calc/surrogate_severe_20200205_036V_regrid.nc
```
I expect @georgemccabe to confirm that this Gen-Ens-Prod difference is a known issue unrelated to these python changes, and that we can proceed with re-creating the MET-11.1.0-beta2 release.

**Comment by JohnHalleyGotway** [comment]
Re-created 11.1.0-beta2 release on 4/28/23.

**Comment by JohnHalleyGotway** [comment]
Re-creating the 11.1.0-beta2 release AGAIN on 5/5/23 to include the fix for issue #2531 which was preventing it from compiling on Hera.
